### PR TITLE
add simple plugins pipeline

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

**Comment by evgeni** [comment]
this is using staging repos, this was easier :woman_shrugging:

**Comment by ehelms** [comment]
This should tell us if all the defined plugins work together but will not necessarily indicate if a single plugin is the problem right? That is if one of the plugins causes a failure, it could be that one or more other plugins would also fail but the code did not hit their failure point yet.

**Comment by ekohl** [comment]
> This should tell us if all the defined plugins work together but will not necessarily indicate if a single plugin is the problem right? That is if one of the plugins causes a failure, it could be that one or more other plugins would also fail but the code did not hit their failure point yet.

Effectively today we test the empty set of plugins. This adds the full set of plugins and then a full set (∅ → F). That is a good test. What you are suggesting is to also test ∅ → {bootdisk} → {bootdisk,discovery} -> ... → F. This is slower and may still not uncover all issues. For example, what if bootdisk is broken. Do you stop testing there? What about all other paths?

There is no support for removing (most) plugins so testing all variations is incredibly expensive. I also expect to find little issues that way. A full set is a good start and at least allows us to write actual tests for the applications. Plugins themselves should already have testing and this is only seeing if our integration all works.

File: pipelines/vars/forklift_plugins.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        - "--enable-foreman-plugin-bootdisk"
+        - "--enable-foreman-plugin-discovery"
+        - "--enable-foreman-plugin-hooks"
+        - "{{ '--enable-foreman-plugin-openscap' if pipeline_os.startswith('centos') else '' }}"
+        - "--enable-foreman-plugin-remote-execution"
+        - "--enable-foreman-plugin-templates"
+        - "--enable-foreman-proxy-plugin-ansible"
+        - "--enable-foreman-proxy-plugin-discovery"
+        - "{{ '--enable-foreman-proxy-plugin-openscap' if pipeline_os.startswith('centos') else '' }}"
+        - "--enable-foreman-proxy-plugin-remote-execution-ssh"
```
[the remark this answers is not in the document]
Comment (evgeni) [reply pipelines/vars/forklift_plugins.yml:26]: maybe, I copied over the definition from luna :)

**Comment by evgeni** [comment]
So running this today, yields https://projects.theforeman.org/issues/31430 -- and yeah, I then SSHed into the box and looked further.

It might be interesting exploring both paths, but ∅ → F is certainly a place we can buld upon I think.

**Comment by ekohl** [comment]
That issue has been reported on Discourse a bunch of times as well. My conclusions from that are that users aren't actually filing bugs and we don't push users to do so. Also that we don't either.

**Review by ekohl — APPROVED** [review]

**Pull request merged**